docs/PREPROCESSING.md

```diff
@@ -64,3 +64,9 @@ src
                ├── get_max_bounding_box.py <- Get bounds of the largest scaled mesh
                └── pc_sdfs.py    <- Sample point clouds from scaled meshes
 ```
+
+The scale factors can be computed using the `get_max_bounding_box` script. Alternatively, the pre-computed scale factors can be downloaded along with the rest of the preprocessed data. The following scale factors are provided 
```

Comment:
Maybe replace "The following scale factors are provided" with "The following scale factors are available for download"